Lesson24/selcan1/index..html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.2/css/all.min.css" integrity="sha512-SnH5WK+bZxgPHs44uWIX+LLJAJ9/2PkPKZ5QiAj6Ta86w+fsb2TkcmfRyVX3pBnMFcV7oQPJkl9QevSCWr3W6A==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <title>Document</title>
+    <style>
+        .kapsayici{
+            width: 100%;;
+            height:100vh;
+            background-color: rgb(60, 60, 104); 
+            display: flex; 
+            justify-content: center;  
+            align-items: center;   
+          }
+        .anaMenu{
+            border-radius: 25px;
+            background-color: white;
+            width: 50%;
+            height: 50%;
+            padding: 30px;
+            display: flex;
+            flex-direction: row;  
+         }
+         @media all and (max-width: 576px) {
+        .kapsayici{
+            background-color: white;
+        }
+         .anaMenu{
+            flex-direction:column-reverse;
```

Comment:
I think on the mobile devices the image should be different, there must be a second file in the folder with files.

Reply:
How will it be?I could'nt understand the second file.

Reply:
Remove second file. You'll have something like this:
```
// CSS
.hidden {
display: none;
}

//HTML
<div id="form-container">
// Form
</div>
<div id="confirmation-container" class="hidden">
// Confirmation
</div>

//JS
//in the event listener for form submit, if the form was filled correctly do something like this:
const formContainer = document.getElementById("form-container");
const confirmationContainer = document.getElementById("confirmation-container");

formContainer.classList.add("hidden");
confirmationContainer.classList.remove("hidden");
```